tools/autograd/templates/Functions.cpp

```diff
@@ -1987,6 +1988,13 @@ Tensor log1p_backward(const Tensor& grad, const Tensor& self) {
   return grad / (self + 1);
 }
 
+// Because the backward of pad(input, pads) is just pad(grad_output, [-p for p in pads])
+Tensor constant_pad_nd_backward(const Tensor& grad, IntList pad) {
+  auto negated_pad = pad.vec();
+  std::transform(negated_pad.cbegin(), negated_pad.cend(), negated_pad.begin(), std::negate<int64_t>());
+  return at::native::constant_pad_nd(grad, negated_pad, 0);
```

Comment:
Thanks! One final nit: Could you change to calling `at::constant_pad_nd` instead? Calling the `native` one will not record `constant_pad_nd ` as a single edge in  backward, and instead record each operation in `at::native:: constant_pad_nd ` an edge, and thus causing double backward to be much slower.